reports/pin/pin.qmd

```diff
@@ -44,9 +45,204 @@ hie_data_subset <- hie_data %>%
+      char_bsmt_fin == "REC" ~ "Finished/formal rec room",
+      char_bsmt_fin == "APT" ~ "Apartment",
+      char_bsmt_fin == "UNF" ~ "Unfinished"
+    ),
+    char_gar1_size = case_when(
+      char_gar1_size == "1CAR" ~ "1 car",
+      char_gar1_size == "1.5CAR" ~ "1.5 cars",
+      char_gar1_size == "2CAR" ~ "2 cars",
+      char_gar1_size == "3CAR" ~ "3 cars",
+      char_gar1_size == "3.5CAR" ~ "3.5 cars",
+      char_gar1_size == "0CAR" ~ "0 cars",
+      char_gar1_size == "4CAR" ~ "4 cars"
+    ),
+    char_ext_wall = case_when(
+      char_ext_wall == "FRAM" ~ "Frame",
+      char_ext_wall == "MASR" ~ "Masonry",
+      char_ext_wall == "FRMA" ~ "Frame/Masonry",
+      char_ext_wall == "STUC" ~ "Stucco"
+    ),
+    char_roof_cnst = case_when(
+      char_roof_cnst == "SHAS" ~ "Shingle/Asphalt",
+      char_roof_cnst == "TRGR" ~ "Tar/Gravel",
+      char_roof_cnst == "SLTE" ~ "Slate",
+      char_roof_cnst == "SHKE" ~ "Shake",
+      char_roof_cnst == "TILE" ~ "Tile",
+      char_roof_cnst == "OTHR" ~ "Other"
+    )
+  )
+
+# Characteristics data table
+# This table contains all the physical attributes of the target PIN
+
+char_table_data %>%
+  select(
+    meta_card_num, char_bldg_sf, loc_school_elementary_district_geoid,
+    meta_class, loc_school_secondary_district_geoid, char_yrblt, char_beds,
+    char_bsmt, char_bsmt_fin, char_fbath, char_hbath, char_porch,
+    char_ext_wall, char_heat, char_air, char_roof_cnst
+  ) %>%
+  mutate(
+    "Improvement Area" = glue("{char_bldg_sf} SqFt", .na = NULL),
+    "Property Class" = char_table_data$meta_class,
+    "Year Built" = as.character(char_table_data$char_yrblt),
+    "Elementary School District" =
+      char_table_data$loc_school_elementary_district_geoid,
+    "Secondary School District" =
+      char_table_data$loc_school_secondary_district_geoid,
+    "Bedrooms" = as.character(char_table_data$char_beds),
+    "Bathrooms" = glue("{char_table_data$char_fbath} Full /
+                       {char_table_data$char_hbath} Half", .na = NULL),
+    "Basement" = glue("{char_table_data$char_bsmt},
+                      {char_table_data$char_bsmt_fin}", .na = NULL),
+    "Garage Size" = char_table_data$char_gar1_size,
+    "Porch" = char_table_data$char_porch,
+    "Facade" = char_table_data$char_ext_wall,
+    "HVAC" = glue("{char_table_data$char_heat},
+                  {char_table_data$char_air}", .na = NULL),
+    "Roof Material" = char_table_data$char_roof_cnst,
+  ) %>%
+  help_clean_tables() %>%
+  replace(is.na(.), "-") %>%
+  gt() %>%
+  tab_options(
+    table.width = pct(100),
+    table.align = "left"
+  ) %>%
+  cols_hide(
+    columns = c(
+      char_bldg_sf, loc_school_elementary_district_geoid, meta_class,
+      loc_school_secondary_district_geoid, char_yrblt, char_beds,
+      char_bsmt, char_bsmt_fin, char_fbath, char_hbath, char_porch,
+      char_ext_wall, char_heat, char_air, char_roof_cnst, meta_card_num
+    )
+  ) %>%
+  tab_footnote(
```

Comment:
**thought:** The chars table itself is fine, but the number of footnotes is pretty wild. I wonder if there's a way for us to restructure this to:

1. Fit all the chars on a table without the need for scrolling
2. Keep the footnotes hidden unless clicked